### DISCUSSION TITLE
Project activity

### DISCUSSION THREAD

**Discussion by Alerinos** [jsakamoto/BlazingStory#1] [Q&A]
Hi, nice project concept. I have some questions about it.

1. Can you do an online demo? So as to make it easier for everyone to understand the design before it is used?
2. Is it only for web components? What about the blazor server version?
3. How do you solve the problem of saving components? Blazor components are compiled together with the project.
4. Do you foresee integration with css frameworks? This is where TailwindCSS would work great.

I'm just thinking about a solution for saving component ustaiweins. You can introduce a "cache base" of settings, so that each component compiles but has "settings", changing them can be done live even on an application that is in production. In addition, you can make groups of settings and even settings of the entire template. With one click you can change the look everywhere.

Thanks for your answer.

**Comment by jsakamoto** [comment]
> 1. Can you do an online demo?

Today, I published a live demonstration site on GitHub Pages.

- https://jsakamoto.github.io/BlazingStory/

It is too simple to impress anyone for now, but I will make it grow up.

> 2. Is it only for web components? What about the blazor server version?

Currently, I verified it only as a Blazor WebAssembly app. However, there is enough possibility to be available for Blazor Server apps too. I'd like to verify it is available for Blazor Server apps in the future. But you can try that right now. ;)

**Reply by Alerinos** [reply]
Cool idea, how do you anticipate recording the changes? 
Do you want to edit the razor file or store the state?
